### PR TITLE
Track split tests in Inspectlet

### PR DESCRIPTION
**Pull request by fauvel**

### CODE REVIEW

**Comment by fauvel** [comment]
@njam please review

File: library/CMService/Inspectlet/Client.php [hunk 1]
```diff
@@ -55,6 +62,24 @@ public function trackPageView(CM_Frontend_Environment $environment, $path) {
...
+
+    /**
+     * @param string $name
+     * @param string $value
+     * @return $this
+     */
+    protected function _addTag($name, $value) {
+        $name = (string) $name;
+        $value = (string) $value;
+        $this->_tagList = (array) $this->_tagList;
```
Comment (njam) [inline library/CMService/Inspectlet/Client.php]: This looks a bit strange. How about initializing the variable to an empty array (in the declaration?), and check for empty array in `getJs()`?

**Comment by fauvel** [comment]
@njam please review

**Comment by fauvel** [comment]
Tagged 1.218.1

**Pull request merged**